tokio/src/runtime/task/harness.rs

```diff
@@ -173,6 +173,10 @@ where
             // the scheduler or `JoinHandle`. i.e. if the output remains in the
             // task structure until the task is deallocated, it may be dropped
             // by a Waker on any arbitrary thread.
+            //
+            // Panics are delivered to the user via the `JoinHandle`. Given that
+            // they are dropping it, we assume they are not interested in it,
+            // and swallow it.
```

Comment:
This replaces some uses of "it" with the actual thing, making it easier to read.
```suggestion
            // they are dropping the `JoinHandle`, we assume they are not
            // interested in the panic and swallow it.
```

Reply:
ooops you are completely right